pallets/weights/src/pallet_identity.rs

```diff
@@ -54,269 +54,364 @@ use polymesh_runtime_common::{RocksDbWeight as DbWeight, Weight};
+    // Proof Skipped: Identity Claims (max_values: None, max_size: None, mode: Measured)
     fn revoke_claim() -> Weight {
-        // Minimum execution time: 44_692 nanoseconds.
-        Weight::from_ref_time(45_033_000)
-            .saturating_add(DbWeight::get().reads(3))
+        // Minimum execution time: 42_668 nanoseconds.
+        Weight::from_ref_time(43_160_000)
+            .saturating_add(DbWeight::get().reads(2))
             .saturating_add(DbWeight::get().writes(1))
     }
     // Storage: Identity KeyRecords (r:1 w:0)
+    // Proof Skipped: Identity KeyRecords (max_values: None, max_size: None, mode: Measured)
     // Storage: Identity Claims (r:1 w:1)
-    // Storage: Asset DisableInvestorUniqueness (r:1 w:0)
-    // Storage: Asset ScopeIdOf (r:1 w:0)
-    // Storage: Asset BalanceOfAtScope (r:1 w:0)
+    // Proof Skipped: Identity Claims (max_values: None, max_size: None, mode: Measured)
     fn revoke_claim_by_index() -> Weight {
-        // Minimum execution time: 48_860 nanoseconds.
-        Weight::from_ref_time(49_221_000)
-            .saturating_add(DbWeight::get().reads(5))
+        // Minimum execution time: 42_911 nanoseconds.
+        Weight::from_ref_time(47_970_000)
+            .saturating_add(DbWeight::get().reads(2))
             .saturating_add(DbWeight::get().writes(1))
     }
     // Storage: Identity KeyRecords (r:2 w:1)
+    // Proof Skipped: Identity KeyRecords (max_values: None, max_size: None, mode: Measured)
     fn set_secondary_key_permissions() -> Weight {
-        // Minimum execution time: 34_264 nanoseconds.
-        Weight::from_ref_time(34_925_000)
+        // Minimum execution time: 42_159 nanoseconds.
+        Weight::from_ref_time(44_141_000)
             .saturating_add(DbWeight::get().reads(2))
             .saturating_add(DbWeight::get().writes(1))
     }
-    /// The range of component `a` is `[0, 1000]`.
-    /// The range of component `p` is `[0, 1000]`.
-    /// The range of component `l` is `[0, 100]`.
-    /// The range of component `e` is `[0, 100]`.
-    fn permissions_cost(a: u32, p: u32, l: u32, e: u32) -> Weight {
-        // Minimum execution time: 200_481 nanoseconds.
-        Weight::from_ref_time(201_151_000)
-            // Manually set for `a`
-            .saturating_add(Weight::from_ref_time(100_000).saturating_mul(a.into()))
-            // Manually set for `p`
-            .saturating_add(Weight::from_ref_time(100_000).saturating_mul(p.into()))
-            // Standard Error: 140_388
-            .saturating_add(Weight::from_ref_time(12_202_214).saturating_mul(l.into()))
-            // Standard Error: 140_388
-            .saturating_add(Weight::from_ref_time(11_817_991).saturating_mul(e.into()))
+    /// The range of component `a` is `[0, 2000]`.
+    /// The range of component `p` is `[0, 2000]`.
+    /// The range of component `l` is `[0, 80]`.
+    /// The range of component `e` is `[0, 80]`.
+    fn permissions_cost(_a: u32, _p: u32, l: u32, e: u32) -> Weight {
```

Comment:
Ignored parameters, but has been addressed in: pallets/common/src/traits/identity.rs by adding this: 
```
if assets > MAX_ASSETS as u32
            || portfolios > MAX_PORTFOLIOS as u32
            || pallets > MAX_PALLETS as u32
            || extrinsics > MAX_EXTRINSICS as u32
        {
            return Weight::MAX;
        }
```

Reply:
The tx fee still needs to be higher for more complex permissions.